pkg/model/keycloak_deployment.go

```diff
@@ -342,6 +314,65 @@ func KeycloakDeploymentReconciled(cr *v1alpha1.Keycloak, currentState *v13.State
 			},
 		},
 	}
-
+	reconciled.Spec.Template.Spec.InitContainers = KeycloakExtensionsInitContainers(cr)
 	return reconciled
 }
+
+func KeycloakExtensionsInitContainers(cr *v1alpha1.Keycloak) []v1.Container {
```

Comment:
These functions should probably be shared between Keycloak and RHSSO. Could you please make them unexported and put them in a separate file? I did similar thing in my backup PR and I used the following filename: `postgresql_common_backup.go`.